Program.cs

```diff
@@ -42,7 +43,7 @@ static void Main(string[] args)
             Process[] potential = Process.GetProcesses();
             foreach (Process process in potential)
             {
-                if (process.ProcessName.StartsWith("PsmDevice"))
+                if (process.ProcessName.StartsWith("PsmDevice") || process.ProcessName.StartsWith("PsmDeviceUnity"))

```

Comment:
Oops, yeah. It's actually 'PsmDeviceForUnity' so just ignore that line. I
quickly added that because it seemed like it wasn't stopping the process.
On Jul 6, 2015 9:18 AM, "Yifan Lu" <notifications@github.com> wrote:

> In Program.cs
> <https://github.com/yifanlu/VitaDefiler/pull/3#discussion_r33932304>:
>
> > @@ -42,7 +43,7 @@ static void Main(string[] args)
> >              Process[] potential = Process.GetProcesses();
> >              foreach (Process process in potential)
> >              {
> > -                if (process.ProcessName.StartsWith("PsmDevice"))
> > +                if (process.ProcessName.StartsWith("PsmDevice") || process.ProcessName.StartsWith("PsmDeviceUnity"))
>
> Doesn't "PsmDeviceUnity" also startswith "PsmDevice"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yifanlu/VitaDefiler/pull/3/files#r33932304>.
>